users/views.py

```diff
@@ -41,3 +40,25 @@ def post(self, request):
 
         except KeyError:
             return JsonResponse({"message": "KEY_ERROR"}, status=400)
+
+
+class SignInView(View):
+    def post(self, request):
+        try:
+            data            = json.loads(request.body)
+            signin_email    = data['email']
+            signin_password = data['password']
+
+            if not User.objects.filter(email = signin_email).exists():
+                return JsonResponse({"message" : "INVALID_UESR"}, status = 401)
+
+            user         = User.objects.get(email = signin_email)
+            access_token = jwt.encode({'user_id' : user.id}, SECRET_KEY, ALGORITHM)
+            
+            if not bcrypt.checkpw(signin_password.encode('utf-8'), user.password.encode('utf-8')):
+                return JsonResponse({"message" : "WRONG PASSWORD"}, status = 401)
```

Comment:
```suggestion
            if not bcrypt.checkpw(signin_password.encode('utf-8'), user.password.encode('utf-8')):
                return JsonResponse({"message" : "INVALID_UESR"}, status = 401)
```
보안을 위해서 아이디가 틀렸는지, 비밀번호가 틀렸는지 모르게 하는 것이 좋습니다.

Reply:
넵 Invalid user 로 변경했습니다.